README.md

```diff
@@ -53,12 +53,54 @@ bearclaw can be run on-demand, or it can rebuild automatically when it sees chan
 - **templates** - header.html and footer.html
 - ***plugins*** *(optional)* - extra features for your page
 
+### How to init a bearclaw
+*As of bearclaw 1.2.0alpha*
+
+#### Linux
+First, download the [bearclaw executable](https://github.com/donuts-are-good/bearclaw/releases)
+
+then create a folder:
+```
+cd your_folder
+```
+
+either have the bearclaw executable in that folder, or set it up in your $PATH, to be able to call `./bearclaw ` from any folder.
+Either way, run the command once to generate the folder structure: :
+
+```
+./bearclaw
+```
+and you get:
+```
+yourbearclawfolder
+├── markdown
+├── output
+│   ├── about.html
+│   └── posts.html
+├── plugins
+└── templates
+```
+
 that's it! point your webserver at `output` or handle it however is best for your case.
 
-**tip:** you can run `bearclaw` and it will run once, or you can use `./bearclaw --watch` to watch the current folder for changes.
+**tip:** you can run `./bearclaw` and it will run once, or you can use `./bearclaw --watch` to watch the current folder for changes.
```

Comment:
Good catch :D bonehead mistake on my part. I see why the Go install issue happened now.